src/Sarif/Sarif.csproj

```diff
@@ -19,7 +19,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461;net45</TargetFrameworks>
+    <!--
+    For compatibility with older clients, we still build Sarid.Sdk (but no other packages)
+    for .NET Framework 4.5.
+    -->
+    <TargetFrameworks>$(TargetFrameworks);net45</TargetFrameworks>
```

Comment:
>net45 [](start = 42, length = 5)

Probably fine to zap this compatibility build moving forward. A good item to throw on the technical debt pile.

Reply:
Will file an infa-debt work item.

---
In reply to: [476588289](https://github.com/microsoft/sarif-sdk/pull/2032#discussion_r476588289) [](ancestors = 476588289)